### PR TITLE
LPS-113598 Deprecate `Liferay.Widget`

### PR DESCRIPTION
**Pull request by julien**
There are currently no usages of `Liferay.Widget` so mark this file as deprecated.

Confirmed this by searching with `git grep -w Liferay.Widget` and `git grep -w liferay-widget`, and used `git log -G'\bLiferay.Widget\b' -p` to determine that the last actual usage was removed in [cad2339833](https://github.com/liferay/liferay-portal/commit/cad233983363608989d1bfc3622775dc26381206).

### CODE REVIEW

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
Screen Shot 2020-05-27 at 13 09 39

Screen Shot 2020-05-27 at 13 09 47

lol

**Comment by wincent** [comment]
Thanks @julien. I'll do a manual forward of this one (seeing as CI is obviously crazy) with a small edit to the description from:

> There are currently no usages of `Liferay.Widget` so mark this file as deprecated.

To include information about *how* we know there are no usages of `Liferay.Widget` (ie. mentioning the steps I imagine you used to verify that this was the case). My motivation here is  trying to make sure our descriptions don't just say "what", but also "why"/"how":

> Confirmed this by searching with `git grep -w Liferay.Widget` and `git grep -w liferay-widget`, and used `git log -G'\bLiferay.Widget\b' -p` to determine that the last actual usage was removed in [cad2339833](https://github.com/liferay/liferay-portal/commit/cad233983363608989d1bfc3622775dc26381206).

Basically, so that anybody reviewing this "remove X because unused" kind of thing can have confidence that it really is unused because you showed them how you arrived at that conclusion, and if they have any doubts, they can re-run the commands and see for themselves. I figure that you spent time doing this in preparing the change, and if you can document it in the commit message and PR description then the review is that much easier and more certain. Also, I am lazy.

**Comment by wincent** [comment]
Manual forward is: https://github.com/brianchandotcom/liferay-portal/pull/89273
